# Generated for NaoQi method ALFileManager.getSystemSharedFolderPath
# Get the current system shared folder path.

---
# System shared folder path string.
string value


